src/main/java/uk/gov/ons/validation/entity/WranglerResponse.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.ons.validation.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class WranglerResponse {
```

Comment:
Would this be a ValidationRequest rather than a WranglerResponse? A response is a reply to the caller whereas this is a request to the next lambda in line.

Reply:
Yes Renamed to ValidationRequest.  Done